config/packages/messenger.yaml

```diff
@@ -0,0 +1,17 @@
+framework:
+    messenger:
+        # reset services after consuming messages
+        reset_on_message: true
+
+        # Uncomment this (and the failed transport below) to send failed messages to this transport for later handling.
```

Comment:
as I understood we need only `reset_on_message: true` here. Can we drop all the rest?